.github/workflows/config.yml

```diff
@@ -0,0 +1,30 @@
+name: main
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements/test.txt
```

Comment:
Let me know if this is the correct txt file. I see there's also a travis one.

Reply:
In Travis, we used Tox. I guess we should use Tox here too: https://github.com/vintasoftware/drf-rw-serializers/blob/master/.travis.yml

See how to do it here: https://github.com/vintasoftware/django-virtual-models/blob/main/.github/workflows/tests.yml